reference/test/matrix/csr_kernels.cpp

```diff
@@ -222,6 +250,40 @@ TEST_F(Csr, ConvertsToCoo)
 }
 
 
+TEST_F(Csr, ConvertsToSellp)
+{
+    auto sellp_mtx = gko::matrix::Sellp<>::create(mtx->get_executor());
+    mtx->convert_to(sellp_mtx.get());
+    assert_equal_to_mtx(sellp_mtx.get());
+}
+
+
+TEST_F(Csr, MovesToSellp)
+{
+    auto sellp_mtx = gko::matrix::Sellp<>::create(mtx->get_executor());
+    auto csr_ref = gko::matrix::Csr<>::create(mtx->get_executor());
+
+    csr_ref->copy_from(mtx.get());
+
```

Comment:
Very small nit:
This newline (between both `csr_ref` calls) should be removed according to our [contributing guidelines](https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines#whitespace) (4th point):
> 4. Unit tests should follow the [AAA](http://wiki.c2.com/?ArrangeActAssert) pattern, and a single blank line must appear between consecutive "A" sections. No other blank lines are allowed in unit tests.